Barotrauma/BarotraumaShared/SharedSource/Items/Components/ItemContainer.cs

```diff
@@ -497,8 +517,11 @@ public override void Update(float deltaTime, Camera cam)
             {
                 SetContainedItemPositions();
             }
-            else if (activeContainedItems.Count == 0)
+            else if (!Inventory.AllItems.Any())
             {
+                item.SendSignal("0", "contained_conditions");
+                item.SendSignal("0", "contained_conditions_percentage");
+                item.SendSignal("0", "contained_items");
```

Comment:
Duplicate code, this and the SendSignal calls done in OnItemRemoved could be consolidated into one method.

Reply:
Made some changes while working on optimization, now the three SendSignal() calls are only made once.